src/app.ts

```diff
@@ -17,9 +17,10 @@ export class App {
     constructor() {
         this.app = express();
         this.port = 8080;
-        this.dbName = 'hello-db';
+        this.dbName = 'tigris_starter_ts';
         this.tigris = new Tigris({
-            serverUrl: 'localhost:8081'
+            serverUrl: 'localhost:8081',
```

Comment:
we should update the starter apps to point to preview in the examples

Reply:
we will discuss this before making change here as there are lots of document refers to local development flow.